fastlane/Fastfile

```diff
@@ -257,6 +257,53 @@ platform :android do
     )
   end
 
+  desc "upload PocketCodeBeta metadata only"
+    lane :upload_Metadata_PocketCodeBeta do
+      upload_to_play_store(
+        package_name:               'org.catrobat.catroid.pocketcodebeta',
+        track:                      'alpha',
+        json_key_data:              ENV['googlePlayJsonKey'],
+        metadata_path:              './fastlane/metadata/android',
+        skip_upload_apk:            'true',
+        skip_upload_metadata:       'false',
+        skip_upload_images:         'false',
+        skip_upload_screenshots:    'false',
+        validate_only:              'false',
+        check_superseded_tracks:    'false',
+      )
+    end
+
+    desc "upload PocketCodeBeta APK to alpha track"
+    lane :upload_APK_PocketCodeBeta do
+      upload_to_play_store(
+        package_name:               'org.catrobat.catroid.pocketcodebeta',
+        track:                      'alpha',
+        json_key_data:              ENV['googlePlayJsonKey'],
+        apk_paths:                  './catroid/build/outputs/apk/pocketcodebeta/signedRelease/catroid-pocketcodebeta-signedRelease.apk',
+        skip_upload_apk:            'false',
+        skip_upload_metadata:       'true',
+        skip_upload_images:         'true',
+        skip_upload_screenshots:    'true',
+        validate_only:              'false',
+        check_superseded_tracks:    'false',
+      )
+    end
+
+    desc "promote PocketCodeBeta APK on alpha track to production"
+    lane :promote_PocketCodeBeta do
+      upload_to_play_store(
+        package_name:               'org.catrobat.catroid.pocketcodebeta',
+        track:                      'alpha',
+        track_promote_to:           'production',
+        json_key_data:              ENV['googlePlayJsonKey'],
+        metadata_path:              './fastlane/metadata/android',
+        skip_upload_apk:            'true',
+        skip_upload_metadata:       'true',
+        skip_upload_images:         'true',
+        skip_upload_screenshots:    'true',
+      )
+    end
```

Comment:
```suggestion
```
Actually, the beta version shall _never_ be promoted to production (because we do not want visible reviews for it, among other reasons), so this part can be deleted.